Quick_Deploy/vLLM/model_repository/vllm/1/model.py

```diff
@@ -167,6 +167,8 @@ async def generate(self, request):
         try:
             request_id = random_uuid()
             prompt = pb_utils.get_input_tensor_by_name(request, "PROMPT").as_numpy()[0]
+            if isinstance(prompt, bytes):
+                prompt = prompt.decode("utf-8")
```

Comment:
Shouldn't need to cast the prompt to str in L189 case?

Reply:
done